packages/orbit-components/src/Modal/index.tsx

```diff
@@ -456,7 +457,7 @@ const Modal = React.forwardRef<Instance, Props>(
                   "[&_.orbit-modal-footer]:duration-fast [&_.orbit-modal-footer]:transition-shadow [&_.orbit-modal-footer]:ease-in-out",
                   fullyScrolled
                     ? "[&_.orbit-modal-footer]:shadow-modal-scrolled"
-                    : "[&_.orbit-modal-footer]:shadow-modal",
+                    : "[&_.orbit-modal-footer]:shadow-modal lm:[&_.orbit-modal-footer]:rounded-b-none",
```

Comment:
If the footer is fixed and the user is scrolling, the footer can't have rounded corners, otherwise the content of the model would be visible from the edges of the rounded corners.